packages/rancher-vsphere-csi/charts/templates/vsphere-csi-node-rbac.yaml

```diff
@@ -0,0 +1,29 @@
+kind: ServiceAccount
```

Comment:
So the DS previously didn't use a service account and now it does ?

Reply:
This is a change from upstream

Reply:
It looks like they were using the default service account and then created their own in a separate manifest. You can see the release notes [here](https://vsphere-csi-driver.sigs.k8s.io/releases/v2.2.0.html), but it doesn’t really say anything about it.